packages/icons/src/leg-icon.js

```diff
@@ -10,11 +10,9 @@ const LegIcon = ({ getCompanyIcon, leg, ModeIcon, ...props }) => {
   if (company && typeof getCompanyIcon === "function") {
     const CompanyIcon = getCompanyIcon(company);
     if (CompanyIcon)
-      return (
-        <Suspense fallback={<span>{company}</span>}>
-          <CompanyIcon {...props} />
-        </Suspense>
-      );
+      <SafeSuspense fallback={<span>{company}</span>}>
```

Comment:
Note that this is for accommodating tests only, and that no release will be triggered for this package.